modules/oe_translation_remote/tests/src/FunctionalJavascript/RemoteTranslationTest.php

```diff
@@ -715,4 +715,55 @@ public function testLanguageCheckboxesSelect(): void {
     $this->assertEquals('Select all', $checkbox->getParent()->find('css', 'label')->getText());
   }
 
+  /**
+   * Tests that the translation sync process throws a proper event subscriber.
```

Comment:
Tests that the translation sync process throws an event which is handled by an event subscriber.